Makefile

```
@@ -304,6 +318,21 @@ uninstall:
 	rm -rf $(DESTDIR)$(INSTALL_BIN)/kplutus
 	rm -rf $(DESTDIR)$(INSTALL_LIB)/kplutus
 
+# Prove tests
+#------------
+KPROVE_OPTS :=
+
+prove_tests := $(wildcard unit-tests/*.md)
```

```suggestion
prove_tests := $(wildcard unit-tests/*-test.md)
```

Depending on what you want, this change will make it so that we're stricter about detecting tests (to not include all `.md` files, but only ones that end in `-test.md`). Up to you.